andrea/js/app.js

```diff
@@ -0,0 +1,67 @@
+// buttons //
+var signUp = document.querySelector(".navbar-sign");
+var logIn = document.querySelector(".navbar-login");
+
+var create_btn = document.querySelector(".create-btn");
+
+// form //
+var form = document.querySelector(".register-form");
+
+// passwords //
+var password = document.getElementById("password");
+var confirm_password = document.getElementById("confirm-password");
+
+// show register form //
+function showRegister() {
+    document.querySelector(".title").style.display = "none";
+    document.querySelector(".description").style.display = "none";
+    signUp.style.display = "none";
+    logIn.style.display = "none";
```

Comment:
create a class called `hidden` and add it to elements you want to hide. You are already using `classList.add` below so lets try to keep using it for the sake of consistency